hub/powertoys/awake.md

```diff
@@ -68,3 +69,8 @@ When setting the value for the `--time-limit` parameter, both of these formats w
 
 - PowerToys.Awake.exe --time-limit 36000
 - PowerToys.Awake.exe --time-limit=36000
+
+When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.parse)
```

Comment:
can we remove the en-us so it is language agnostic.

```suggestion
When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](https://learn.microsoft.com/dotnet/api/system.datetime.parse)
```

Reply:
Indeed, this needs to be locale-agnostic.

Reply:
Done - we want all links inside learn.ms to be relative links as well.